deploy/clowdapp.yaml

```diff
@@ -247,3 +249,6 @@ parameters:
 - name: REST_WORKERS
   description: The number of workers in Sanic's REST API
   value: "2"
+- name: ENVIRONMENT_NAME
+  description: The name of the environment. Needs to be either stage or prod
+  value: "stage"
```

Comment:
Should we default to `prod` so if something goes wrong it will be prod when not set